### PR TITLE
chore: update the js v5 page

### PR DESCRIPTION
**Pull request by Ngozi-Txfusion**
### What

- **Updated the JS v5 Page:**
  - Revised the content and structure of the JavaScript v5 page to reflect the latest changes and improvements in the SDK, ensuring it provides accurate and up-to-date information for developers.

- **Changed the Feel and Appearance:**
  - Modified the design and layout of the documentation to enhance user experience, making it more visually appealing and easier to navigate.

### Why

- **JS v5 Page Updates:**
  - To provide developers with the most current information and guidance on using JavaScript v5, ensuring that the documentation aligns with the latest SDK version.

- **Feel and Appearance Changes:**
  - To improve the overall user experience by making the documentation more attractive and user-friendly, which helps developers find the information they need more efficiently and enjoyably.

### CODE REVIEW

File: content/sdk/10.js/00.ethers/10.v5/07.utils.md [hunk 1]
```diff
@@ -644,8 +656,9 @@ const isValidSignature = await utils.isMessageSignatureCorrect(provider, ADDRESS
 
 ### `isSignatureCorrect`
 
-Called from [`isMessageSignatureCorrect`](#ismessagesignaturecorrect)
-and [`isTypedDataSignatureCorrect`](#istypeddatasignaturecorrect). Returns true if account abstraction EIP712
+Called from [`isMessageSignatureCorrect`](/sdk/js/ethers/v5/utils#ismessagesignaturecorrect)
+and [`isTypedDataSignatureCorrect`](/sdk/js/ethers/v5/utils#istypeddatasignaturecorrect). Returns true if
```
Comment (itsacoyote) [inline content/sdk/10.js/00.ethers/10.v5/07.utils.md:660]:
```suggestion
and [`isTypedDataSignatureCorrect`](#istypeddatasignaturecorrect). Returns true if
```

File: content/sdk/10.js/00.ethers/10.v5/07.utils.md [hunk 2]
```diff
@@ -844,4 +857,6 @@ const l1ContractAddress = utils.undoL1ToL2Alias(l2ContractAddress);
 // const l1ContractAddress = "0x702942B8205E5dEdCD3374E5f4419843adA76Eeb"
 ```
 
-See also [`applyl1tol2alias`](#applyl1tol2alias).
+::callout
+See also [`applyl1tol2alias`](/sdk/js/ethers/v5/utils#applyl1tol2alias).
```
Comment (itsacoyote) [inline content/sdk/10.js/00.ethers/10.v5/07.utils.md:861]:
```suggestion
See also [`applyl1tol2alias`](#applyl1tol2alias).
```

**Review by itsacoyote — CHANGES_REQUESTED** [review]

File: content/sdk/10.js/00.ethers/10.v5/01.features.md [hunk 3]
```diff
@@ -1,99 +1,110 @@
...
 }
 ```
 
 ## Encoding paymaster params
 
-While the paymaster feature by itself does not impose any limitations on values of the
-`paymasterInput`, the Matter Labs team endorses certain types of
-[paymaster flows](https://docs.zksync.io/build/developer-reference/account-abstraction.html#built-in-paymaster-flows)
+While the paymaster feature itself does not impose any limitations on values of the `paymasterInput`, the Matter Labs
+team endorses certain types of [paymaster flows](https://docs.zksync.io/build/developer-reference/account-abstraction.html#built-in-paymaster-flows)
```
Comment (itsacoyote) [inline content/sdk/10.js/00.ethers/10.v5/01.features.md:97]:
```suggestion
team endorses certain types of [paymaster flows](https://docs.zksync.io/build/developer-reference/account-abstraction/paymasters#built-in-paymaster-flows)
```

File: content/sdk/10.js/00.ethers/10.v5/01.features.md [hunk 4]
```diff
@@ -1,99 +1,110 @@
...
+The ZKsync SDK provides a utility method that can be used to get the correctly formed `paymasterParams` object: [getPaymasterParams](/sdk/js/ethers/v5/paymaster-utils#getpaymasterparams).
 
-## See in action
+## Using a paymaster with a contract method
 
-If you want to call the method `setGreeting` of an ethers `Contract` object called `greeter`,
-this would look the following way, while paying fees with the
-[testnet paymaster](https://docs.zksync.io/build/developer-reference/account-abstraction.html#testnet-paymaster):
+If you want to call the `setGreeting` method of an ethers `Contract` object called `greeter`, this would look the
+following way, while paying fees with the [testnet paymaster](https://docs.zksync.io/build/developer-reference/account-abstraction.html#testnet-paymaster):
```
Comment (itsacoyote) [inline content/sdk/10.js/00.ethers/10.v5/01.features.md:105]:
```suggestion
following way, while paying fees with the [testnet paymaster](https://docs.zksync.io/build/developer-reference/account-abstraction/paymasters#testnet-paymaster):
```

File: content/sdk/10.js/00.ethers/10.v5/03.providers.md [hunk 5]
```diff
@@ -1,38 +1,35 @@
...
 
-Two providers are available:
+## Available providers
 
-- [`Provider`](#provider): Supplies the same functionality as
-  [`ethers.providers.JsonRpcProvider`](https://docs.ethers.org/v5/api/providers/jsonrpc-provider/#JsonRpcProvider)
-  and extends it with ZKsync-specific methods.
-- [`Web3Provider`](#web3provider): Extends the ZKsync Era [`Provider`](#provider) class to make it more compatible
-  with Web3 wallets.
+- **Provider**: Extends [`ethers.JsonRpcProvider`](https://docs.ethers.org/v5/api/providers/jsonrpc/#about-jsonrpcProvider)
```
Comment (itsacoyote) [inline content/sdk/10.js/00.ethers/10.v5/03.providers.md:14]: It looks like a variety of Ethers docs links are broken, please go through and check that they're up to date.

```suggestion
- **Provider**: Extends [`ethers.JsonRpcProvider`](https://docs.ethers.org/v5/api/providers/jsonrpc-provider/)
```

File: content/sdk/10.js/00.ethers/10.v5/05.accounts-l1-l2.md [hunk 6]
```diff
@@ -1,46 +1,47 @@
...
+tags: ["zksync", "account management", "wallet creation", "transaction signing", "cryptocurrency", "ethereum", "l1 to l2 transactions"]
 ---
 
-This section explores the methods which allow the [account](/sdk/js/ethers/v5/accounts) to send transactions
-among both L1 to L2 networks.
+Learn how to send transactions between L1 and L2 networks using the `zksync-ethers` library.
 
-If you want some background on how L1<->L2 interaction works on ZKsync, go through the
-[introduction](https://docs.zksync.io/build/developer-reference/l1-l2-interop.html).
+If you need background information on how L1<->L2 interactions work on ZKsync, check out the [Developer Reference](https://docs.zksync.io/build/developer-reference/l1-l2-interop.html).
```
Comment (itsacoyote) [inline content/sdk/10.js/00.ethers/10.v5/05.accounts-l1-l2.md:9]:
```suggestion
If you need background information on how L1<->L2 interactions work on ZKsync, check out the [Developer Reference](https://docs.zksync.io/build/developer-reference/l1-l2-interoperability).
```

**Review by itsacoyote — CHANGES_REQUESTED** [review]
You may need to check through the URLs to make sure they're not broken. We also made some changes to URLs for docs.zksync.io. They should redirect to a page, but they might not be the correct up to date page.

**Review by petarTxFusion — DISMISSED** [review]

File: content/sdk/10.js/00.ethers/10.v5/02.front-end.md [hunk 7]
```diff
@@ -1,16 +1,17 @@
...
+If your front-end does not deploy new smart contracts, you don't need to change your codebase! All existing SDKs and
+infrastructure will work seamlessly.
 
-## Enabling ZKsync features
+## Using ZKsync Era features
 
-If you want to deploy smart contracts or enable advanced ZKsync features, like account abstraction,
-then you need to use the `zksync-ethers` library. You can read about the basics [here](/sdk/js/ethers/v5/features).
+To deploy smart contracts or use advanced ZKsync features like Account Abstraction, you'll need the `zksync-ethers`
+library. Learn the basics of `zksync-ethers` in the [ZKsync Era Features](/sdk/js/ethers/v6/features).
```
Comment (itsacoyote) [inline content/sdk/10.js/00.ethers/10.v5/02.front-end.md:17]:
```suggestion
library. Learn the basics of `zksync-ethers` in the [ZKsync Era Features](/sdk/js/ethers/v5/features).
```

File: content/sdk/10.js/00.ethers/10.v5/03.providers.md [hunk 8]
```diff
@@ -314,7 +311,7 @@ async getAllAccountBalances(address: Address): Promise<BalancesMap>
 
 #### Example
 
-Helper function: [toJSON](#tojson).
+Helper function: [toJSON](/sdk/js/ethers/v5/providers#tojson).
```
Comment (itsacoyote) [inline content/sdk/10.js/00.ethers/10.v5/03.providers.md:314]:
```suggestion
Helper function: [toJSON](#tojson).
```

**Review by itsacoyote — CHANGES_REQUESTED** [review]

**Review by danijelTxFusion — APPROVED** [review]

**Review by itsacoyote — APPROVED** [review]

**Pull request merged**